ref/general/health-check-microservices.adoc

```diff
@@ -25,15 +25,15 @@ A self-check can assess anything that the service needs:
 
-For example, in a microservices-based banking application, health checks might be implemented on the login service, the balance transfer service, and the bill pay service. If a health check on the balance transfer service detects a bug or deadlock, it reports as DOWN. In this case, if the `/health/live` endpoint is configured in a Kubernetes liveness probe, the probe restarts the service automatically, saving the user from encountering downtime or an error and preserving the functions of the other services in the application.
+For example, in a microservices-based banking application, you might implement health checks on the login service, the balance transfer service, and the bill pay service. If a health check on the balance transfer service detects a bug or deadlock, it reports as DOWN. In this case, if the `/health/live` endpoint is configured in a Kubernetes liveness probe, the probe restarts the service automatically. This feature saves the user from encountering downtime or an error and preserves the functions of the other services in the application.
 
 == MicroProfile Health provided endpoints and annotations
 
-MicroProfile Health provides three endpoints:
+MicroProfile Health provides the following three endpoints:
 
 - `/health/ready`: returns the readiness of a service, or whether it is ready to process requests. This endpoint corresponds to the Kubernetes readiness probe.
-- `/health/live`: returns the liveness of a service, or whether it has encountered a bug or deadlock. If this check fails, the service is not running and can be terminated. This endpoint corresponds to the Kubernetes liveness probe, which automatically restarts the pod if the check fails.
-- `/health`: aggregates the responses from `/health/live` and `/health/ready`. This endpoint corresponds to the deprecated `@Health` annotation and is only available to provide compatibility with MicroProfile 1.0. Developers who are using MicroProfile 2.0 or higher should use the `/health/ready` or `/health/live` endpoints instead.
+- `/health/live`: returns the liveness of a service, or whether it encountered a bug or deadlock. If this check fails, the service is not running and can be stopped. This endpoint corresponds to the Kubernetes liveness probe, which automatically restarts the pod if the check fails.
+- `/health`: aggregates the responses from `/health/live` and `/health/ready`. This endpoint corresponds to the deprecated `@Health` annotation and is only available to provide compatibility with MicroProfile 1.0. Developers who are using MicroProfile 2.0 or higher can use the `/health/ready` or `/health/live` endpoints instead.
```

Comment:
"should" is the right word here. We don't want developers who are able to use MP2.0 using the old endpoints. We can't say "must" because that would raise the question of "or else what?". Alternatively rephrase the sentence to "If you are using MP 2 or higher, use the ....endpoints instead." (actually I prefer that)